slave-base/contrib/bin/run-jnlp-client

```diff
@@ -34,6 +34,28 @@ fi
 
 echo "OPENSHIFT_JENKINS_JVM_ARCH='${OPENSHIFT_JENKINS_JVM_ARCH}', CONTAINER_MEMORY_IN_MB='${CONTAINER_MEMORY_IN_MB}', using $(readlink /etc/alternatives/java) and $(readlink /etc/alternatives/javac)"
 
+if [[ -z "${NOOP_NO_PROXY_DEFAULT}" ]]; then
```

Comment:
let's call this "SKIP_NO_PROXY_DEFAULT" just because noop_no is hard for my brain to process :)